bigbluebutton-html5/.eslintrc.js

```diff
@@ -24,6 +24,8 @@ module.exports = {
     'react/prop-types': 1,
     'jsx-a11y/no-access-key': 0,
     'react/jsx-props-no-spreading': 'off',
+    'max-classes-per-file': ['error', 2],
+    'no-param-reassign': ['error', { props: false }],
```

Comment:
Which line/file was triggering this param-reassign warning?

Reply:
This one: https://github.com/bigbluebutton/bigbluebutton/blob/e8d59ed14acfb2784c6015e58f47436042b5da2a/bigbluebutton-html5/imports/ui/components/muted-alert/component.jsx#L82

Reply:
I believe i can rewrite this with deconstructor, so we don't need to disable param-reassign (although i did it for props only).

Reply:
I restricted the scope for now, so we don't need to change it in .eslint file.